sdk/tests/conformance2/rendering/blitframebuffer-multisampled-readbuffer.html

```diff
@@ -140,27 +140,43 @@
     gl.enableVertexAttribArray(positionLoc);
     // Draw two triangles. They are almost right-triangles: one is actually an obtuse triangle. The other is actually an acute triangle.
     // This means that some egdes of the triangles almost parallel with x-axis and y-axis. So they can show the alias.
-    var positions = new Float32Array([
+    /* var positions = new Float32Array([
          0.85, -0.90,
         -0.90,  0.85,
         -0.87, -0.87,
          0.90, -0.85,
          0.93,  0.93,
         -0.85,  0.90,
+    ]);*/
+    var positions = new Float32Array([
+         1.0, -1.0,
+        -1.0,  1.0,
+        -1.0, -1.0,
+         1.0, -1.0,
+         1.0,  1.0,
+        -1.0,  1.0,
     ]);
     var positionBuffer = gl.createBuffer();
     gl.bindBuffer(gl.ARRAY_BUFFER, positionBuffer);
     gl.bufferData(gl.ARRAY_BUFFER, positions, gl.STATIC_DRAW);
     gl.vertexAttribPointer(positionLoc, 2, gl.FLOAT, false, 0, 0);
 
     gl.enableVertexAttribArray(colorLoc);
-    var colors = [
+    /* var colors = [
         [255, 0, 0, 255],
         [0, 255, 0, 255],
         [0, 0, 255, 255],
         [253, 122, 5, 255],
         [6, 252, 120, 255],
         [118, 4, 254, 255]
+    ];*/
+    var colors = [
+        [255, 0, 0, 255],
+        [0, 255, 0, 255],
+        [0, 0, 255, 255],
+        [255, 0, 0, 255],
+        [0, 0, 255, 255],
+        [0, 255, 0, 255],
```

Comment:
Good suggestion, let's fill a rectangle with solid color and draw to a multisampled srgb draw buffer. It is simple and direct.